tests/subsys/bluetooth/fast_pair/storage/src/main.c

```diff
@@ -313,4 +251,8 @@ void test_main(void)
 			 );
 
 	ztest_run_test_suite(fast_pair_storage_tests);
+	/* Test cases verifying that module properly handles settings data corruption.
+	 * Corrupted data test suite relies on internal data representation.
+	 */
+	test_corrupted_data_run();
```

Comment:
It's somehow convoluted. Couldn't we have a separate project for this test category?

Reply:
The mocked settings are shared by both test suites. Also both test suites are used to test `fp_storage` subsystem.
I would prefer to keep them together because of that. The main reason why I created a separate test suite is the fact that I wanted to limit visibility of fp_storage_priv only to file that actually uses it.